beacon-chain/p2p/discovery.go

```diff
@@ -454,11 +453,7 @@ func peersFromStringAddrs(addrs []string) ([]ma.Multiaddr, error) {
 }
 
 func multiAddrFromString(address string) (ma.Multiaddr, error) {
-	addr, err := iaddr.ParseString(address)
-	if err != nil {
-		return nil, err
-	}
-	return addr.Multiaddr(), nil
+	return ma.NewMultiaddr(address)
```

Comment:
What's up with this, my not change then calls to `multiAddrFromString`?

Reply:
github.com/ipfs/go-ipfs-addr is deprecated and no longer works with new updates to `github.com/multiformats/go-multiaddr`

Reply:
I see what you are saying as feedback though. There is no longer a need to have this helper function.

Let's follow up refactor this in another PR to minimize changes.